Automattic/wp-calypso
# Github Deployments: Remove guard

The GitHub deployments feature is behind a feature check, after its launch, we can now remove all the unnecessary guards

## Proposed Changes

* Removed the feature check and we always load the GitHub Deployments command.
* Removed the feature check on the site's ellipsis menu.

## Testing Instructions



* Open the Calypso live link
* Press CMD + K
* Search for `Github`
* The GitHub deployments command should be visible.
* Navigate to the global navigation
* Press the ellipsis menu on a hosting-enabled site.
* You should see the GitHub deployments item.

GitHub deployments command palette
![ghdeploy-remove](https://github.com/Automattic/wp-calypso/assets/1989914/601ed219-1563-4c1a-b34a-db04ec674e17)

GitHub deployment sites ellipsis
![ghdeploy-ellipsis](https://github.com/Automattic/wp-calypso/assets/1989914/de00b5ea-6496-4aad-85dd-ddf3e4461945)



## Pre-merge Checklist
